content/applications/productivity/discuss/chatter.rst

```diff
@@ -0,0 +1,416 @@
+   date can be changed by selecting a day on the calendar in the :guilabel:`Due Date` field.
+
+Lastly, click one of the following buttons:
+
+- :guilabel:`Schedule`: adds the activity to the chatter under :guilabel:`Planned activities`.
+- :guilabel:`Mark as Done`: adds the details of the activity to the chatter under :guilabel:`Today`.
+  The activity is not scheduled, it is automatically marked as completed.
+- :guilabel:`Done \& Schedule Next`: adds the task under :guilabel:`Today` marked as done, and
+  opens a new activity window.
+- :guilabel:`Discard`: discards any changes made on the pop-up window.
+
+Scheduled activities are added to the chatter for the record under :guilabel:`Planned activities`,
+and are color-coded based on their due date.
+
+- **Red** icons indicate an overdue activity.
+- **Yellow** icons indicate an activity with a due date scheduled for the current date.
+- **Green** icons indicate an activity with a due date scheduled in the future.
+
+.. image:: chatter/chatter-activity-icons.png
+   :align: center
+   :alt: A chatter thread with planned activities with varying due dates.
+
+.. tip::
+   Click the :guilabel:`ℹ️ (info)` icon next to a planned activity to see additional details.
+
+   .. image:: chatter/planned-activity-details.png
+      :align: center
+      :alt: A detailed view of a planned activity.
+
+After completing an activity, click :guilabel:`Mark Done` under the activity entry in the chatter.
+This opens a :guilabel:`Mark Done` pop-up window where additional notes about the activity can be
+entered. After adding any comments to the pop-up window click: :guilabel:`Done \& Schedule Next`,
+:guilabel:`Done`, or :guilabel:`Discard`.
+
+After the activity is marked complete, an entry with the activity type, title, and any details that
+were included in the pop-up window is listed in the chatter.
+
+.. image:: chatter/chatter-completed-activity.png
+   :align: center
+   :alt: A chatter thread with a completed activity, included additional details.
+
+.. _discuss/attach-files:
+
+Attach files
+============
+
+Files can be added as attachments in chatter, either to send with messages, or to include with a
+record.
+
+.. note::
+   After a file has been added to a chatter thread, it can be downloaded by any user with access to
+   the thread. Click the :guilabel:`📎 (paperclip)` icon to make the files header visible, if
+   necessary. Then, click the download icon on the file to download it.
+
+To attach a file, click the :guilabel:`📎 (paperclip)` icon located at the top of the chatter
+composer of any record that contains a chatter thread.
+
+This opens a file explorer pop-up window. Navigate to the desired file, select it, then click
+:guilabel:`Open` to add it to the record. Alternatively, files can be dragged and dropped directly
+onto a chatter thread.
+
+After files have been added, they are listed in the chatter thread, under a *Files* heading.
```

Comment:
```suggestion
After files have been added, they are listed in the chatter thread, under a :guilabel:`Files` heading.
```
opting for a GUI here because that's how it appears/reads on the screen/UI